lib/emoji.js

```diff
@@ -32,6 +38,18 @@ Emoji.prototype._get = function _get(emoji) {
 };
 
 /**
+ * get string emoji code for string emoji
+ * @param  {string} emoji
+ * @return {string}
+ */
+Emoji.prototype._getStringEmoji = function _getStringEmoji(emoji) {
```

Comment:
This code is mostly the same too, if the added `:` in `_get`s return statement is a problem, you can pass a second argument (boolean) that will adjust the output accordingly.